tools/coq_dune.ml

```diff
@@ -214,7 +214,7 @@ let record_dune d ff =
   if Sys.file_exists sd && Sys.is_directory sd then
     let out = open_out (bpath [sd;"dune"]) in
     let fmt = formatter_of_out_channel out in
-    if List.nth d 0 = "plugins" then
+    if List.nth d 0 = "stdlib" && Sys.file_exists (bpath [sd;"plugin_base.dune"]) then
       fprintf fmt "(include plugin_base.dune)@\n";
```

Comment:
When does plugin_base exist but we don't want to include it? ie why the list.nth part of the test?

Reply:
That's just a hack, `coq_dune` should go away next week anyways so I wouldn't worry too much about this.